jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/dialects/postgresql/PostgreSQLTableMetadataReader.java

```diff
@@ -0,0 +1,86 @@
+package com.exasol.adapter.dialects.postgresql;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.List;
+import java.util.Optional;
+
+import com.exasol.adapter.AdapterProperties;
+import com.exasol.adapter.dialects.PostgreSQLIdentifierMapping;
+import com.exasol.adapter.jdbc.*;
+import com.exasol.adapter.metadata.TableMetadata;
+
+/**
+ * This class handles the specifics of mapping PostgreSQL table metadata to Exasol
```

Comment:
Missing dot at the end of first line of Javadoc is (personally) annoying but maybe javadoc already complains :)

Reply:
Added.